apps/passport-server/src/database/queries/telegram/insertTelegramConversation.ts

```diff
@@ -19,3 +19,21 @@ on conflict do nothing;`,
   );
   return result.rowCount;
 }
+
+export async function insertTelegramEvent(
```

Comment:
we don't test the telegram db functions at all afaik. could you update `devconnectdb.spec.ts` with some basic tests that cover these the telegram functions, along with the other telegram db functions we have?